packages/libs/pactjs-test-project/src/example-templates/generated-tx-templates/index.ts

```diff
@@ -2,114 +2,114 @@
 import { buildUnsignedTransaction, ICommandBuilder } from '@kadena/client';
 
 export default {
-  "safe-transfer": (args: {
-    "fromAcct": string,
-    "toAcct": string,
-    "amount": string,
-    "chain": string,
-    "network": string,
-    "fromKey": string,
+  'safe-transfer': (args: {
```

Comment:
still this annoying issue

Reply:
haha..normally I revert the change..I forgot